src/types/stardust/block/address/mod.rs

```diff
@@ -66,29 +66,47 @@ impl From<Address> for Bson {
     }
 }
 
-#[cfg(test)]
+#[cfg(feature = "rand")]
+mod rand {
+    use super::*;
+
+    impl Address {
+        /// Generates a random alias [`Address`].
+        pub fn rand_alias() -> Self {
+            Self::Alias(AliasAddress::rand())
+        }
+
+        /// Generates a random nft [`Address`].
+        pub fn rand_nft() -> Self {
+            Self::Nft(NftAddress::rand())
+        }
+
+        /// Generates a ed25519 [`Address`].
+        pub fn rand_ed25519() -> Self {
+            Self::Ed25519(Ed25519Address::rand())
+        }
+    }
+}
+
+#[cfg(all(test, feature = "rand"))]
```

Comment:
I think here we should only have `#[cfg(test)]`, whereas `#[cfg(any(test, feature = "rand"))]` should go on `mod rand`.

_Notice the `any` in the `cfg` statement above._

I think in it's current form, we would not be able to run any tests without the `rand` feature.

Reply:
> I think in it's current form, we would not be able to run any tests without the rand feature.

Yes, because they all depend on random data. That's kind of the point. This way we can write tests that don't depend on random data without the extra feature gate.